### PR TITLE
Added HomePage Screens and Folder Structure

### PR DESCRIPTION
**Pull request by manojasarsa**
This PR includes Screens for  ->
- Home page screens
- Folder Structure
- NavBar
- Header
- logo
- Search box
- Wishlist icon with a badge
- Cart icon with a badge
- Profile icon
- Login button

### CODE REVIEW

File: node_modules/.bin/yarn [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+case `uname` in
+    *CYGWIN*|*MINGW*|*MSYS*) basedir=`cygpath -w "$basedir"`;;
+esac
+
+if [ -x "$basedir/node" ]; then
+  exec "$basedir/node"  "$basedir/../yarn/bin/yarn.js" "$@"
+else 
+  exec node  "$basedir/../yarn/bin/yarn.js" "$@"
+fi
```
Comment (KoushikKintali) [inline node_modules/.bin/yarn]: you should not push node_modules folder in github

File: e-commerce-store/src/components/index.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export {Header} from "./Header/Header";
+export {Category} from "./Category/Category";
+export {ShopNow} from "./ShopNow/ShopNow";
+export {HeroBanner} from "./HeroBanner/HeroBanner";
```
Comment (poornima-alugubelly) [inline e-commerce-store/src/components/index.js:4]: nice use of barrel export

**Review by poornima-alugubelly — APPROVED** [review]
I like how u structured ur code

File: node_modules/.bin/yarn (hunk 1 above)
Comment (manojasarsa) [reply node_modules/.bin/yarn]: Oh is it so? I didn't knew that. Thanks for feedback!

**Pull request merged**